didntreally.CodeReviews.Console.Calculator/Calculator/Program.cs

```diff
@@ -0,0 +1,179 @@
+                    int userNum = 0;
+                    while (!int.TryParse(userString, out userNum))
+                    {
+                        Console.WriteLine("This is not a valid number, please enter a number! ");
+                        userString = Console.ReadLine();
+                    }
+
+                    calculations.RemoveAt(userNum-1);
+
+                    
+                    if (calculations.Count() != 0)
+                    {
+                        Console.WriteLine($"{userNum} has been removed, the new list is ");
+                        for (int x = 0; x < calculations.Count(); x++)
+                        {
+                            Console.WriteLine($"Your past results are Number {x + 1} : {calculations[x]}");
+
+                        }
+
+                        Console.WriteLine("Do you still want to continue using calculator? Type y for yes / n for no");
+                        string userContinue = Console.ReadLine().ToUpper();
+
+                        while(String.IsNullOrWhiteSpace(userContinue) || double.TryParse(userContinue, out ifInputIsInt) || (!yesOrNo.Contains(userContinue))){
+
+                            Console.WriteLine("Please enter a correct alphabet y/n");
+                            userContinue = Console.ReadLine().ToUpper();
+                        }
+
+
+                        if (userContinue == "Y")
+                        {
+                            runCalculator = true;
+                        }
+                        else if(userContinue == "N"){
+                            runCalculator = false;
+                            Console.WriteLine("Thanks for using our Calculator!");
+                        }
+                        
+                        
+                    }
+                    else
+                    {
+                        Console.WriteLine("The list is empty now!");
+                        Console.WriteLine("Do you still want to continue using calculator? Type y for yes / n for no");
+                        string userContinue2 = Console.ReadLine().ToUpper();
+
+                        while (String.IsNullOrWhiteSpace(userContinue2) || double.TryParse(userContinue2, out ifInputIsInt) || !yesOrNo.Contains(userContinue2))
+                        {
+
+                            Console.WriteLine("Please enter a correct alphabet y/n");
+                            userContinue2 = Console.ReadLine().ToUpper();
+                        }
+
+
+                        if (userContinue2.ToUpper() == "Y")
+                        {
+                            runCalculator = true;
+                        }
+                        else if (userContinue2.ToUpper() == "N")
+                        {
+                            runCalculator = false;
+                            Console.WriteLine("Thanks for using our Calculator!");
+                        }
+                    }
+                    
```

Comment:
Removed unnecessary empty lines. I like this habit to cease early in someone's code career. You want your cultivate tidiness in your code, it will stand out in to prospect employers. 😊